### PR TITLE
[wip]Post upgrade

### PR DESCRIPTION
**Pull request by nohaibogdan1**

### CODE REVIEW

File: imports/ui/pages/Posts/PostEdit.jsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ export default class PostEdit extends React.Component {
     }
 
     submit = (post) => {
+        console.log(post);
```
Comment (thisUsernameAlreadyExists) [inline imports/ui/pages/Posts/PostEdit.jsx]: remove

File: imports/ui/pages/Posts/schema.js [hunk 2]
```diff
@@ -12,5 +12,8 @@ export default new SimpleSchema({
     },
     title: {
         type: String
+    },
+    visibleForEveryone: {
```
Comment (thisUsernameAlreadyExists) [inline imports/ui/pages/Posts/schema.js]: change name

File: imports/ui/pages/Posts/schema.js [hunk 3]
```diff
@@ -12,5 +13,10 @@ export default new SimpleSchema({
...
         type: String
+    },
+    isVisibleForEveryone: {
+        type: Boolean,
+        defaultValue: true
     }
-});
\ No newline at end of file
+});
+
```
Comment (thisUsernameAlreadyExists) [inline imports/ui/pages/Posts/schema.js]: remove extra line

File: imports/db/posts/schema.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import SimplSchema from 'simpl-schema';
-import PostTypesEnum from './enums/types';
 import _ from 'underscore';
 
+import PostTypesEnum from './enums/types';
+
+
```
Comment (thisUsernameAlreadyExists) [inline imports/db/posts/schema.js]: remove extra line

File: imports/api/reactions/reactionService.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import {Meteor} from 'meteor/meteor';
+
+import Security from '/imports/api/security';
+import {Reactions} from '/imports/db';
+import listReactionsQuery from '/imports/api/reactions/queries/listReactions';
+
+export default class ReactionService {
+    static updateReaction(reaction) {
+        Security.checkLoggedIn(reaction.userId);
```
[the remark this answers is not in the document]
Comment (nohaibogdan1) [reply imports/api/reactions/reactionService.js]: An unregistered user can't react to a post

File: imports/db/posts/enums/types.js [hunk 6]
```diff
@@ -7,4 +9,4 @@ const PostTypesEnum = {
     OTHERS: 'others'
 };
 
-export default PostTypesEnum;
\ No newline at end of file
+export const types = _.values(PostTypesEnum);
```
Comment (thisUsernameAlreadyExists) [inline imports/db/posts/enums/types.js]: const types = _.values(PostTypesEnum);

export {you variables}

File: imports/db/posts/schema.js [hunk 7]
```diff
@@ -12,8 +12,8 @@ export default new SimplSchema({
     },
     postType: {
         type: String,
-        allowedValues: _.values(PostTypesEnum),
-        defaultValue: _.values(PostTypesEnum)[0]
+        allowedValues: types,
+        defaultValue: types[0]
```
Comment (thisUsernameAlreadyExists) [inline imports/db/posts/schema.js]: you can import PostTypesEnum from post type enums and use it here

File: imports/db/reactions/reactions.enum.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+const PostReactionsEnum = {
+    LIKE: 'like',
+    LOVE: 'love',
+    HAPPY: 'happy',
+    WOW: 'wow',
+    SAD: 'sad',
+    ANGRY: 'angry'
+}
+
+export const reactions = _.values(PostReactionsEnum);
```
Comment (thisUsernameAlreadyExists) [inline imports/db/reactions/reactions.enum.js]: export {your variables}

File: imports/db/reactions/schema.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import SimpleSchema from 'simpl-schema';
+
+
```
Comment (thisUsernameAlreadyExists) [inline imports/db/reactions/schema.js]: remove extra space

File: imports/ui/pages/Posts/PostElement.jsx [hunk 10]
```diff
@@ -2,15 +2,45 @@ import React from 'react';
...
 import PropTypes from 'prop-types';
 
+
 import RoutesEnum from '/imports/ui/routes/enums/routes';
 import generateRoutes from '/imports/ui/routes/methods';
 
+import ReactionButton from './ReactionButton';
+import {reactions} from '/imports/db/reactions/reactions.enum';
+
+
```
Comment (thisUsernameAlreadyExists) [inline imports/ui/pages/Posts/PostElement.jsx]: remove extra line

File: imports/ui/pages/Posts/schema.js [hunk 11]
```diff
@@ -1,16 +1,21 @@
...
 export default new SimpleSchema({
     description: {
         type: String
     },
     postType: {
         type: String,
-        allowedValues: _.values(PostTypesEnum),
-        defaultValue: _.values(PostTypesEnum)[0]
+        allowedValues: types,
+        defaultValue: types[0]
```
Comment (thisUsernameAlreadyExists) [inline imports/ui/pages/Posts/schema.js]: import postTypes the use it as default

File: imports/db/posts/schema.js (hunk 7 above)
Comment (nohaibogdan1) [reply imports/db/posts/schema.js]: Why cannot types?

File: imports/ui/pages/Posts/schema.js (hunk 11 above)
Comment (nohaibogdan1) [reply imports/ui/pages/Posts/schema.js]: Why cannot use types?

**Pull request merged**